src/core/mdn-annotation.js

```diff
@@ -189,3 +171,33 @@ function getMdnKey(conf) {
   if (typeof mdn === "string") return mdn;
   return mdn.key || shortName;
 }
+
+/**
+ * @param {string} key MDN key
+ * @param {object} mdnConf
+ * @param {string} [mdnConf.specMapUrl]
+ * @param {string} [mdnConf.baseJsonPath]
+ * @param {number} [mdnConf.maxAge]
+ *
+ * @typedef {{ [browser in keyof MDN_BROWSERS]: { version_added: string } }} MdnSupportEntry
```

Comment:
```suggestion
 * @typedef {Record<keyof MDN_BROWSERS, { version_added: string }>} MdnSupportEntry
```

Reply:
I kinda prefer this to `Record<>`, as we can specify what the key is (`browser` as a string here).

Reply:
Well, I think `keyof MDN_BROWSERS` already is a useful hint in this case 😉

Reply:
Good point!